src/report/display.rs

```diff
@@ -13,6 +13,7 @@ impl ResultName for FailureReason {
             FailureReason::Unknown => "failed (unknown)".into(),
             FailureReason::Timeout => "timed out".into(),
             FailureReason::NetworkAccess => "network access".into(),
+            FailureReason::Docker => "docker".into(),
```

Comment:
Nit: can we have a slightly better message here? This would result in "build docker" as the outcome, which is puzzling. Would be better to have "failed (docker error)" as the message here, to have "build failed (docker error)" as the outcome.

Reply:
That makes sense similar to unknown is `failed (unknown)` rather than just `unknown`

Reply:
Ok, I have pushed a commit fixing this.